mkosi/__init__.py

```diff
@@ -2119,7 +2103,7 @@ def bump_image_version(uid: Optional[int] = None, gid: Optional[int] = None) ->
         logging.info(f"Increasing last component of version by one, bumping '{version}' → '{new_version}'.")
 
     Path("mkosi.version").write_text(f"{new_version}\n")
-    if uid and gid:
+    if uid is not None and gid is not None:
```

Comment:
I wondered whether this had an edge case where `uid` or `gid` could be non-`None` and `os.chown` would be called with one `None` value, which would throw an error, but this is caught by an assert at the beginning of the function (why is that a requirement anyway?). Why don't we make the default in the signature `-1` instead of `None` and type this as an `int`. Passing `-1` will make the chown a noop and we don't have to have this check anymore.